### PR TITLE
docs: add info about e2e tests

### PR DESCRIPTION
**Pull request by asimpson**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -36,6 +36,13 @@ You can also have `yarn` watch for changes and automatically recompile them:
 yarn watch
 ```
 
+#### Running E2E tests
+
+1. Install [K6](https://k6.io/docs/get-started/installation/)
+1. Ensure Grafana is running on port `:3000`
+1. Ensure clickhouse server is running on port `:9000`, hint you can use the `docker-compose.yml` file to get the needed services running.
```
Comment (mshustov) [inline CONTRIBUTING.md:43]: Maybe we should add a docker-compose command to copy?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -36,6 +36,13 @@ You can also have `yarn` watch for changes and automatically recompile them:
 yarn watch
 ```
 
+#### Running E2E tests
+
+1. Install [K6](https://k6.io/docs/get-started/installation/)
```
Comment (mshustov) [inline CONTRIBUTING.md:41]: It's also can be add as a part of docker-compose

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -36,6 +36,13 @@ You can also have `yarn` watch for changes and automatically recompile them:
...
+1. Ensure Grafana is running on port `:3000`
+1. Ensure clickhouse server is running on port `:9000`, hint you can use the `docker-compose.yml` file to get the needed services running.
+1. Run `k6 run e2e/e2ek6.test.js`
```
Comment (mshustov) [inline CONTRIBUTING.md:44]: Let's add it as an npm script, which is a standard way in the JS community. to distribute built-in scripts

**Review by SpencerTorres — COMMENTED** [review]
I agree with @mshustov that there should be an npm script, maybe `yarn run test:e2e` or similar. It looks like there's already a script command called `server` for starting docker compose, but I think that info belongs in its own section.

File: CONTRIBUTING.md (hunk 2 above)
Comment (asimpson) [reply CONTRIBUTING.md:41]: k6 in docker introduces a lot of complexity, for now I'm content with having k6 being a requirement. In the future we can lever the dagger ci work to remove this step.

File: CONTRIBUTING.md (hunk 3 above)
Comment (asimpson) [reply CONTRIBUTING.md]: done.

**Comment by asimpson** [comment]
Good feedback @mshustov and @SpencerTorres, I've updated the `package.json` to spin up docker-compose and then run k6 tests.

**Review by mshustov — APPROVED** [review]

**Review by aangelisc — APPROVED** [review]

**Review by SpencerTorres — APPROVED** [review]

**Pull request merged**